scripts/docgen/docgen.php

```diff
@@ -187,6 +187,20 @@ function save_file($filename, $content) { /* {{{ */
 }
 /* }}} */
 
+function get_type_of_value($value) { /* {{{ */
+    if (is_int($value)) {
+        return 'int';
```

Comment:
Oh, right, I'd also seen that `&integer;` was recommended over `<type>int</type>` in review comments when I submitted PRs that used the output of this function. I should do that instead

Reply:
Hmm, I think so far we prefer the entities only for "free text", but the synopsis use explicit `<type>` elements. It might be okay to change that (although likely a lot of work for translators), but at the very least we need to coordinate with gen_stub.php which [generates `<type>` elements](https://github.com/php/php-src/blob/ef5ed06d3cc615ad3ae5c7b7d601ab9e4a35111e/build/gen_stub.php#L719-L741).

@kocsismate, thoughts?

Reply:
Yeah, I'd appreciate if docgen generated code which is compatible with the output of gen_stub.